### DISCUSSION TITLE
开源社区治理与运营的 mindmap

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#100] [日常讨论]
根据最近的开源治理规范与标准（社区端），整理的一个图~

![image](https://user-images.githubusercontent.com/15010826/170846702-4818285d-5784-4118-9806-a890341ec58a.png)

**Comment by wangleo61** [comment]
王老师点赞，两个小补充哈，在B01是否可以添加：安全规范，在B03是否可以添加：合规风险：然后包括：EAR合规，许可证合规，知识产权合规，商标合规，著作权合规


Leo

发件人: Will ***@***.***>
发送时间: Sunday, May 29, 2022 8:11 AM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] 开源社区治理与运营的 mindmap (Discussion #100)


根据最近的开源治理规范与标准（社区端），整理的一个图~

[image]<https://user-images.githubusercontent.com/15010826/170846674-eb639cab-8979-482f-a555-011624b42490.png>

―
